minio-client/runtime/src/main/java/io/quarkiverse/minio/client/MinioClients.java

```diff
@@ -83,11 +95,48 @@ public MinioClient createMinioClient(String minioClientName) {
         return builder.build();
     }
 
-    private MinioRuntimeConfiguration getConfiguration(String minioClientName) {
+    public MinioAsyncClient createMinioAsyncClient(String minioClientName) {
+        MinioRuntimeConfiguration configuration = getAndVerifyConfiguration(minioClientName);
```

Comment:
Can we avoid the duplication ? Not sure how though, as there is no common interface.

Reply:
Yeah, I checked the Minio Client source. I'm open for any solution better than this :sweat_smile: